condensation/cloud.py

```diff
@@ -337,7 +337,7 @@ def migrate_group(self, cloud, strict=True):
             group_to_migrate, cloud)
         if flavors_left:
             if strict:
-                msg = "cannot fit flavors " + flavors_left
+                msg = "cannot fit flavors %s" % flavors_left
```

Comment:
Flavor IDs as ints are fixed in https://github.com/MirantisWorkloadMobility/CloudFerry/pull/206.